lib/main.js

```diff
@@ -10,25 +10,41 @@ import eik from '@eik/core';
 import config from './config.js';
 import * as utils from './utils.js';
 
+/**
+ * @typedef {object} EikServiceOptions
+ * @property {import('@eik/sink').default} [sink]
+ * @property {import('@eik/sink').default} [customSink] [Deprecated] Use sink instead
+ * @property {string} [aliasCacheControl]
+ * @property {string} [notFoundCacheControl="public, max-age=5"]
+ */
+
 const EikService = class EikService {
-    constructor({
-        customSink,
-        notFoundCacheControl,
-        aliasCacheControl,
-    } = {}) {
+    /**
+     * @param {EikServiceOptions} [options={}]
+     */
+    constructor(options = {}) {
```

Comment:
If we decide to keep the "bin-option", then I assume this would cause issues for what is happening here: https://github.com/eik-lib/service/blob/deprecate-built-in-sinks/bin/eik-server.js#L6

But I guess that is perhaps for the actual deprecation pr 😄

Reply:
It's optional now, so you can new up an instance without passing any opts 👍